src/content/structured/components/tooltips/code.mdx

```diff
@@ -26,57 +26,93 @@ import { IcTooltip, IcButton } from "@ukic/react";
+<ic-tooltip target="text-button" label="This is a description of the button" placement="bottom"></ic-tooltip>`,
   },
   {
     language: "React",
-    snippet: `<IcTooltip target="text-button" label="This is a description of the button" placement="left" />
-<IcTooltip target="text-button" label="This is a description of the button" placement="top" />
-<IcTooltip target="text-button" label="This is a description of the button" placement="bottom" />
-<IcTooltip target="text-button" label="This is a description of the button" placement="right" />`,
+    snippet: `<IcTooltip target="text-button" label="This is a description of the button" placement="top" />
+<IcTooltip target="text-button" label="This is a description of the button" placement="bottom" />`,
   },
 ];
 
-<ComponentPreview style={{ gap: "8px" }} snippets={snippets}>
-  <IcTooltip
-    label="This is a description of the button"
-    target="test-button"
-    placement="left"
-  >
-    <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
-      Left
-    </IcButton>
-  </IcTooltip>
-  <IcTooltip
-    label="This is a description of the button"
-    target="test-button"
-    placement="top"
-  >
-    <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
-      Top
-    </IcButton>
-  </IcTooltip>
-  <IcTooltip
-    label="This is a description of the button"
-    target="test-button"
-    placement="bottom"
-  >
-    <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
-      Bottom
-    </IcButton>
-  </IcTooltip>
-  <IcTooltip
-    label="This is a description of the button"
-    target="test-button"
-    placement="right"
-  >
-    <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
-      Right
-    </IcButton>
-  </IcTooltip>
+<ComponentPreview snippets={snippets}>
+  <div style={{ display: "flex", flexDirection: "column", gap: "8px" }}>
+    <IcTooltip
+      label="This is a description of the button"
+      target="test-button"
+      placement="top"
+    >
+      <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
+        Top
+      </IcButton>
+    </IcTooltip>
+    <IcTooltip
+      label="This is a description of the button"
+      target="test-button"
+      placement="bottom"
+    >
+      <IcButton id="test-button" aria-describedby="ic-tooltip-test-button">
+        Bottom
+      </IcButton>
+    </IcTooltip>
+  </div>
 </ComponentPreview>
 
 <ComponentDetails component="ic-tooltip" />
+
+## Variants
+
+### Left placement
+
+export const snippetsLeft = [
+  {
+    language: "Web component",
+    snippet: `<ic-tooltip target="text-button" label="This is a description" placement="left"></ic-tooltip>`,
+  },
+  {
+    language: "React",
+    snippet: `<IcTooltip target="text-button" label="This is a description" placement="left" />`,
+  },
+];
+
+<ComponentPreview style={{ marginLeft: "45vw" }} snippets={snippetsLeft}>
```

Comment:
The buttons go outside the component preview at larger screen sizes:
![image](https://user-images.githubusercontent.com/111882109/204534350-9e78e70c-caae-479b-afc6-aad8c110ac0a.png)
I think it's because of the 'vw' - might need to be 'px' rather than a unit that's relative to the viewport